README.md

```diff
@@ -147,14 +147,34 @@ $('.login-dbconn').click(function () {
   });
 ```
 
-### Passwordless authentication with SMS
+### Passwordless authentication
 
-First you must activate and configure your passwordless [Twilio](https://twilio.com) connection in our [dashboard](https://manage.auth0.com/#/connections/passwordless).
+Passwordless authentication allows users to login without credentials, by using an email or a SMS message, which can be delivered through `startPasswordless()`.
```

Comment:
`...without credentials` does not fell right.

`without the need to set a fixed/recurrent password` or something like that.

I'm neither convinced with my proposal :dancer: